GitCommands/Utils/EnvUtils.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Runtime.InteropServices;
 using Microsoft.Win32;
 
 namespace GitCommands.Utils
 {
     public static class EnvUtils
     {
+        [DllImport("user32.dll")]
+        private static extern bool IsIconic(IntPtr handle);
```

Comment:
We define interops in the UI layer, have a look at GitUI/Interop.

Reply:
I added Interops, should probably be added to GitCommands instead (Interops only in GitUI now)